scipy-1.0/paper.tex

```diff
@@ -1184,28 +1177,30 @@ \section*{Supporting Information}
 \setcounter{figure}{0}
 \setcounter{table}{0}
 
-\begin{figure}[H]
```

Comment:
Not certain that this needs to be removed entirely, but should it be a whole page? Suggest to match style with other figures.